sw.js

```diff
@@ -1 +1,4 @@
-// empty for now
+self.importScripts('/assets/js/vendor/workbox-sw.js')
```

Comment:
you should include the version version here or add a revision hash otherwise you may not be able to update the library